### ISSUE TITLE
An alternative text-to-multi-view generator 

### ISSUE THREAD

**Issue by Xiangyu1Sun** [autonomousvision/LaRa#15]
Dear author, thanks for your great work. 
For text-to-multi-view generator, are there anything could alternative Instant3D?

**Comment by apchenstu** [comment]
Yes, thanks! Instantmesh provides a single view to a multi-view model, it's worth trying.
